keps/sig-cli/kustomize-file-processing-integration.md

```diff
@@ -77,7 +79,7 @@ and provide a path for addressing issues around error handling and messaging.
 
 Integrate kustomize directly into libraries that enable file processing for cli-runtime (e.g. resource builder).
 Kubectl commands taking the common flags (`-f`, `--filename`, `-R`, `--recursive`) will support `kustomization.yaml`
-files.
+files through a new flag `-k` or `--kustomization`.
```

Comment:
`--kustomization` here, `--kustomize` below?

Reply:
The latter is shorter, go with it.